src/App/Components/Redirection/redirect.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import queryString from 'query-string';
+
+const style = {
+  textAlign: 'center',
```

Comment:
@MCFrank16  I think it would be cool to move these styles to a sass file.

Reply:
Ohh yeah I forgot to move them